Calculate/Calculate/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculate
+{
+    public class Program
+    {
+        public static void Main(string[] args) 
+        {
+            Program prog = new();
```

Comment:
```suggestion
            Program program = new();
```
Avoid abbreviations and acronyms in variable names